### PR TITLE
Upgrade Protractor

### PR DESCRIPTION
**Pull request by alecmerdler**
### Description

Bumps Protractor to the latest minor version to remove a hack for configuring logging level thanks to [this PR](https://github.com/angular/protractor/pull/4068).

### CODE REVIEW

**Comment by sosiouxme** [comment]
@alecmerdler please include the following diffs to the PR for updating downstream deps:

```
--- a/sources
+++ b/sources
@@ -1,2 +1,2 @@
 401aa1438604b8e32306fa2bd4f8e211  node-v8.9.4-headers.tar.gz
-60d04378ee7fe27c3ac97b6a6bd3d65b  yarn-offline.tar
+8ae6d0c3b9d049f081d8482d566bf343  yarn-offline.tar
```

File: sources [hunk 1]
```diff
@@ -1,2 +1,2 @@
 401aa1438604b8e32306fa2bd4f8e211  node-v8.9.4-headers.tar.gz
-60d04378ee7fe27c3ac97b6a6bd3d65b  yarn-offline.tar
+8ae6d0c3b9d049f081d8482d566bf343  yarn-offline.tar
```
Comment (alecmerdler) [inline sources]: @sosiouxme Modified `sources` as requested.

**Comment by spadgett** [comment]
jenkins rebuild

**Comment by alecmerdler** [comment]
jenkins rebuild

**Pull request merged**